src/libsync/CMakeLists.txt

```diff
@@ -109,13 +110,13 @@ add_library(${synclib_NAME} SHARED ${libsync_SRCS})
 target_link_libraries(${synclib_NAME}
     "${csync_NAME}"
     ${OS_SPECIFIC_LINK_LIBRARIES}
-    Qt5::Core Qt5::Network
+    Qt5::Core Qt5::Network Qt5::Widgets
```

Comment:
Eh, wait!  we don't want to add more widget dependency in libsync.

Well, then we no longer need TOKEN_AUTH_ONLY.

(We used to have customer project that needed to work without QtGui,  but these are no longer existing at the moment,  still would be good to keep linsync working withuout gui)

Reply:
Hm I'd expect that tokenauth only means also no oauth? in that case I'll just move the QtWidget back into the if